components/accordion/index.css

```diff
@@ -218,124 +262,78 @@
-
-	min-block-size: var(--mod-accordion-min-block-size, var(--spectrum-accordion-min-block-size));
+	/* start spacing controlled by edge to disclosure icon spacing */
+	padding-inline: 0 var(--mod-accordion-edge-to-content-area, var(--spectrum-accordion-edge-to-content-area));
 	line-height: var(--mod-accordion-item-header-line-height, var(--spectrum-accordion-item-header-line-height));
 
 	text-overflow: ellipsis;
-	cursor: pointer;
+	cursor: var(--spectrum-accordion-item-header-cursor);
 	font-size: var(--mod-accordion-item-header-font-size, var(--spectrum-accordion-item-header-font-size));
 	font-weight: var(--mod-accordion-item-header-font-weight, var(--spectrum-accordion-item-header-font-weight));
 	font-style: var(--mod-accordion-item-header-font-style, var(--spectrum-accordion-item-header-font-style));
 	font-family: var(--mod-accordion-item-header-font, var(--spectrum-accordion-item-header-font));
 
-	/* reset styling if button element is used */
+	/* Reset styling of button element */
 	appearance: none;
 	border: 0;
 	text-align: start;
 	inline-size: 100%;
 
-	&:focus {
-		outline: none;
-
-		&::after {
-			content: "";
-			position: absolute;
-			inset-inline-start: 0;
-		}
-	}
 	color: var(--mod-accordion-item-header-color-default, var(--spectrum-accordion-item-header-color-default));
 	background-color: var(--mod-accordion-background-color-default, var(--spectrum-accordion-background-color-default));
 
-	&:hover {
+	&:hover &:not(.is-disabled) {
 		color: var(--mod-accordion-item-header-color-hover, var(--spectrum-accordion-item-header-color-hover));
 		background-color: var(--mod-accordion-background-color-hover, var(--spectrum-accordion-background-color-hover));
-
-		+ .spectrum-Accordion-itemIconContainer {
-			color: var(--mod-accordion-item-header-color-hover, var(--spectrum-accordion-item-header-color-hover));
-		}
 	}
 
-	&:focus-visible {
+	&:focus-visible &:not(.is-disabled) {
 		border-radius: var(--mod-accordion-corner-radius, var(--spectrum-accordion-corner-radius));
-
-		outline: var(--mod-accordion-focus-indicator-thickness, var(--spectrum-accordion-focus-indicator-thickness)) solid var(--mod-accordion-focus-indicator-color, var(--spectrum-accordion-focus-indicator-color));
 		background-color: var(--mod-accordion-background-color-key-focus, var(--spectrum-accordion-background-color-key-focus));
 		color: var(--mod-accordion-item-header-color-key-focus, var(--spectrum-accordion-item-header-color-key-focus));
-		outline-offset: calc(var(--mod-accordion-focus-indicator-gap, var(--spectrum-accordion-focus-indicator-gap)) * -1);
+		outline: var(--mod-accordion-focus-indicator-thickness, var(--spectrum-accordion-focus-indicator-thickness)) solid var(--mod-accordion-focus-indicator-color, var(--spectrum-accordion-focus-indicator-color));
+		outline-offset: var(--mod-accordion-focus-indicator-gap, var(--spectrum-accordion-focus-indicator-gap));
```

Comment:
`--spectrum-accordion-focus-indicator-gap` is the actual token, so should we be creating a similar custom property to use in our styles, instead of directly using the token? I thought that was our preference, unless that's changed.